cdp_scrapers/instances/portland.py

```diff
@@ -239,41 +321,67 @@ def get_votes(self, event_page: BeautifulSoup) -> Optional[List[Vote]]:
         # Clearly “Yea” -> VoteDecision.APPROVE, but we don’t have enough information
         # for what words to map to other constants.
 
-        return reduced_list(
-            [
-                self.get_none_if_empty(
-                    Vote(decision=None, person=self.get_person(None))
-                ),
-            ],
+        vote_element_uncle = minute_section.find(
+            "div", text=re.compile("Votes"), attrs={"class": "field__label"}
         )
+        if vote_element_uncle is None:
+            return None
+        vote_element_parent = vote_element_uncle.next_sibling
+        vote_elements = vote_element_parent.find_all("div", class_="relation--type-")
+        vote_list = []
+        for vote_element in vote_elements:
+            vote = vote_element.text.strip()
+
+            i = vote.rfind(" ")
+            decision = vote[i:].strip()
+            if decision == "Yea":
+                decision = VoteDecision.APPROVE
+            elif decision == "Nay":
+                decision = VoteDecision.REJECT
+            elif decision == "Absent":
+                decision = VoteDecision.ABSENT_NON_VOTING
+            else:
+                decision = None
+            name = self.separate_name_from_title(vote[0:i].strip())
+            vote_list.append(
+                self.get_none_if_empty(
+                    Vote(decision=decision, person=self.get_person(name))
+                )
+            )
 
-    def get_event_minutes(
-        self, event_page: BeautifulSoup
-    ) -> Optional[List[EventMinutesItem]]:
+        return reduced_list(vote_list)
+
+    def get_event_minutes(self, event_page: Tag) -> Optional[List[EventMinutesItem]]:
```

Comment:
Actually Shak this should stay as `event_page: BeautifulSoup`. In [this line](https://github.com/Shak2000/cdp-scrapers/blob/3643a2f6a389fdb6caf26d11b3b9032839e6baf1/cdp_scrapers/instances/portland.py#L476) you are calling into `get_event_minutes()` like

```Python
event_minutes_items=self.get_event_minutes(event_page.soup),
```

Reply:
Done.